README.md

```diff
@@ -1165,6 +1166,15 @@ $ git log --committer=<name or email>
 
 Keep in mind that author and committer are not the same. The `--author` is the person who originally wrote the code; on the other hand, the `--committer`, is the person who committed the code on behalf of the original author.
 
+<a name="i-want-to-find-a-tag-where-a-commit-is-referenced"></a>
+### I want to find a tag where a commit is referenced
+
+To find all tags containing a specific commit
```

Comment:
I think this might be a better title than 'find a tag where a commit is referenced'. It is clearer.